src/components/Atoms/AccountSwitcher.jsx

```diff
@@ -0,0 +1,157 @@
+import cx from 'classnames'
+import React, {
+  createContext,
+  useContext,
+  useEffect,
+  useRef,
+  useState
+} from 'react'
+import { Outlet, useLocation, useNavigate } from 'react-router-dom'
+import { buildAccountsQuery } from 'src/queries'
+
+import { BarComponent, BarCenter } from 'cozy-bar'
+import { useClient, useQuery } from 'cozy-client'
+import Button from 'cozy-ui/transpiled/react/Buttons'
+import Chip from 'cozy-ui/transpiled/react/Chips'
+import DropdownButton from 'cozy-ui/transpiled/react/DropdownButton'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import CalendarIcon from 'cozy-ui/transpiled/react/Icons/Calendar'
+import CheckboxIcon from 'cozy-ui/transpiled/react/Icons/Checkbox'
+import ExchangeIcon from 'cozy-ui/transpiled/react/Icons/Exchange'
+import People from 'cozy-ui/transpiled/react/Icons/People'
+import PieChartIcon from 'cozy-ui/transpiled/react/Icons/PieChart'
+import WalkIcon from 'cozy-ui/transpiled/react/Icons/Walk'
+import { Layout, Main, Content } from 'cozy-ui/transpiled/react/Layout'
+import List from 'cozy-ui/transpiled/react/List'
+import ListItemIcon from 'cozy-ui/transpiled/react/ListItemIcon'
+import Menu from 'cozy-ui/transpiled/react/Menu'
+import Nav, {
+  NavItem,
+  NavIcon,
+  NavText,
+  genNavLink
+} from 'cozy-ui/transpiled/react/Nav'
+import Sidebar from 'cozy-ui/transpiled/react/Sidebar'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import Alerter from 'cozy-ui/transpiled/react/deprecated/Alerter'
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+import MenuItem from 'cozy-ui/transpiled/react/MenuItem'
```

Comment:
You can use `ActionsMenu` instead of a simple `Menu` so it display a bottomDrawer on mobile to select an option ([doc](https://docs.cozy.io/cozy-ui/react/#/Extra?id=actionsmenu))

Reply:
I moved to `ActionsMenu` in the same commit -> https://github.com/cozy/cozy-papillon-app/pull/29/commits/d36b4d535e5a9c0b648836dfef4bde9d91d7f531